intro-HPC/ch_introduction.tex

```diff
@@ -103,14 +103,57 @@ \section{What is the \hpcInfra?}
 \fi
 \ifleuven
   \begin{enumerate}
-    \item  112 compute nodes with 2 10-core Ivy Bridge processors, 64 GB memory,
-           300 GB local disk, QDR-IB network
-    \item  32 compute nodes with 2 10-core Ivy Bridge processors, 128 GB memory,
-           300 GB local disk, QDR-IB network
-    \item  8 nodes with 2 8-core Sandy Bridge processors, 64 GB memory,
-           300 GB local disk, DDR-IB network, 2 K20 NVIDIA GPGPUs
-    \item  8 nodes with 2 8-core Sandy Bridge processors, 64 GB memory,
-           300 GB local disk, DDR-IB network, 2 Intel Xeon Phis
+     \item ThinKing (thin node cluster)
+      \begin{itemize}
+       \item  176 compute nodes with 2 10-core Ivy Bridge processors, 64 GB memory,
+              300 GB local disk, QDR-IB network
+       \item  32 compute nodes with 2 10-core Ivy Bridge processors, 128 GB memory,
+              300 GB local disk, QDR-IB network
+       \item  48 compute nodes with 2 12-core Ivy Bridge processors, 64 GB memory,
+              300 GB local disk, QDR-IB network
+      \end{itemize}
+    \item Accelerator partition
+      \begin{itemize}
+       \item  8 nodes with 2 8-core Sandy Bridge processors, 64 GB memory,
+              300 GB local disk, DDR-IB network, 2 K20 NVIDIA GPGPUs
+       \item  8 nodes with 2 8-core Sandy Bridge processors, 64 GB memory,
+              300 GB local disk, DDR-IB network, 2 Intel Xeon Phis
+       \item  4 nodes with 2 6-core Xeon 5650 Westmere processors, 24 GB memory, 
+              300 GB local disk, DDR-IB network, 2 Tesla M2070 GPGPUs
+       \item  5 nodes with 2 10-core Haswell Xeon E5-2650v3 processors, 64 GB memory, 
+              300 GB local disk, DDR-IB network, 2 Tesla K40 GPGPUs
+      \end{itemize}
+    \item Visualization partition
+      \begin{itemize}
+       \item  2 nodes with 2 10-core Haswell Xeon E5-2650v3 processors, 128 GB memory, 
+              300 GB local disk, DDR-IB network, 2 NVIDIA Quadro K5200 GPUs (visualization nodes)    
+      \end{itemize}
+    \item Cerebro (shared memory SMP section)
+      \begin{itemize}
+        \item  64 sockets with 10-core Xeon E5-4650 processors, total 14 TB memory, 
+               SGI-proprietary NUMAlink6 interconnect, 
+               1 partition has 480 cores and 12 TB RAM and another partition has 160 cores and 2 TB RAM, 
+               both partitions have 10TB local scratch space
+      \end{itemize}
+%    \item  176 compute nodes with 2 10-core Ivy Bridge processors, 64 GB memory,
+%           300 GB local disk, QDR-IB network
```

Comment:
It can be removed. I commented it out as later I decided to split clusters with enumerate first.

From: Kenneth Hoste [mailto:notifications@github.com]
Sent: donderdag 18 februari 2016 14:04
To: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com>
Cc: Mag Selwa <mag.selwa@kuleuven.be>
Subject: Re: [vsc_user_docs] Feature/leuven chapter1 (#106)


In intro-HPC/ch_introduction.tex<https://github.com/hpcugent/vsc_user_docs/pull/106#discussion_r53310299>:

> +              300 GB local disk, DDR-IB network, 2 Tesla K40 GPGPUs

> +      \end{itemize}

> +    \item Visualization partition

> +      \begin{itemize}

> +       \item  2 nodes with 2 10-core Haswell Xeon E5-2650v3 processors, 128 GB memory,

> +              300 GB local disk, DDR-IB network, 2 NVIDIA Quadro K5200 GPUs (visualization nodes)

> +      \end{itemize}

> +    \item Cerebro (shared memory SMP section)

> +      \begin{itemize}

> +        \item  64 sockets with 10-core Xeon E5-4650 processors, total 14 TB memory,

> +               SGI-proprietary NUMAlink6 interconnect,

> +               1 partition has 480 cores and 12 TB RAM and another partition has 160 cores and 2 TB RAM,

> +               both partitions have 10TB local scratch space

> +      \end{itemize}

> +%    \item  176 compute nodes with 2 10-core Ivy Bridge processors, 64 GB memory,

> +%           300 GB local disk, QDR-IB network

remove this commented out part?

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/106/files#r53310299>.